app/javascript/packages/document-capture/higher-order/with-background-encrypted-upload.jsx

```diff
@@ -3,17 +3,17 @@ import UploadContext from '../context/upload';
 import AnalyticsContext from '../context/analytics';
 
 /**
- * Returns a promise resolving to an DataView representation of the given Blob object.
+ * Returns a promise resolving to an ArrayBuffer representation of the given Blob object.
```

Comment:
Based on the "an" here, I seem to recall that I'd originally implemented this as an `ArrayBuffer` conversion, and per ba5bb1e549c92a25d0729148e0149b67c5598325 had only decided to apply a `DataView` wrapper due to issues with our test environment dependencies which have since been resolved upstream.